autobuild.xml

```diff
@@ -2870,36 +2844,128 @@ Copyright (c) 2012, 2014, 2015, 2016 nghttp2 contributors</string>
       <map>
         <key>platforms</key>
         <map>
-          <key>common</key>
+          <key>darwin64</key>
           <map>
             <key>archive</key>
             <map>
               <key>hash</key>
-              <string>8e365eff8dcace48d91e2530f8b13e420849aefc</string>
-              <key>hash_algorithm</key>
-              <string>sha1</string>
+              <string>8cff2060843db3db788511ee34a8e8cc</string>
               <key>url</key>
-              <string>https://github.com/secondlife/3p-vulkan-gltf-pbr/releases/download/v1.0.0-d7c372f/vulkan_gltf-1.0.0-common-d7c372f.tar.zst</string>
+              <string>https://automated-builds-secondlife-com.s3.amazonaws.com/ct2/101316/891509/vulkan_gltf-1-darwin64-572743.tar.bz2</string>
             </map>
             <key>name</key>
-            <string>common</string>
+            <string>darwin64</string>
+          </map>
+          <key>windows</key>
+          <map>
+            <key>archive</key>
+            <map>
+              <key>hash</key>
+              <string>58eea384be49ba756ce9c5e66669540b</string>
+              <key>url</key>
+              <string>https://automated-builds-secondlife-com.s3.amazonaws.com/ct2/101318/891520/vulkan_gltf-1-windows-572743.tar.bz2</string>
+            </map>
+            <key>name</key>
+            <string>windows</string>
+          </map>
+          <key>windows64</key>
+          <map>
+            <key>archive</key>
+            <map>
+              <key>hash</key>
+              <string>79b6a11622c2f83cfc2b7cd1fafb867b</string>
+              <key>url</key>
+              <string>https://automated-builds-secondlife-com.s3.amazonaws.com/ct2/101319/891521/vulkan_gltf-1-windows64-572743.tar.bz2</string>
+            </map>
+            <key>name</key>
+            <string>windows64</string>
           </map>
         </map>
         <key>license</key>
         <string>Copyright (c) 2018 Sascha Willems</string>
         <key>license_file</key>
-        <string>vulkan_gltf.txt</string>
+        <string>LICENSES/vulkan_gltf.txt</string>
         <key>copyright</key>
         <string>Copyright (c) 2018 Sascha Willems</string>
         <key>version</key>
-        <string>1.0.0</string>
+        <string>1</string>
         <key>name</key>
         <string>vulkan_gltf</string>
         <key>canonical_repo</key>
         <string>https://bitbucket.org/lindenlab/3p-vulkan-gltf-pbr</string>
         <key>description</key>
         <string>Vulkan GLTF Sample Implementation</string>
       </map>
+      <key>webrtc</key>
+      <map>
+        <key>platforms</key>
+        <map>
+          <key>darwin64</key>
+          <map>
+            <key>archive</key>
+            <map>
+              <key>creds</key>
+              <string>github</string>
```

Comment:
creds should not be here. it will lead to errors unless one sets `AUTOBUILD_GITHUB_TOKEN`
``` 
Package webrtc is set to use 'github' credentials type but no AUTOBUILD_GITHUB_TOKEN environment variable is set
ERROR: Failed to download package 'webrtc' from 'https://github.com/secondlife/3p-webrtc-build/releases/download/m114.5735.08.59-debug/webrtc-m114.5735.08.59-debug.8823062977-windows64-8823062977.tar.zst'
```